sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/DDLParserSuite.scala

```diff
@@ -646,19 +646,6 @@ class DDLParserSuite extends AnalysisTest {
         Some(first())))
   }
 
-  test("alter table: update column type, comment and position") {
-    comparePlans(
-      parsePlan("ALTER TABLE table_name CHANGE COLUMN a.b.c " +
-        "TYPE bigint COMMENT 'new comment' AFTER d"),
```

Comment:
This now matches HIVE style alter column:
```
ALTER TABLE table [PARTITION partition_spec]
   CHANGE [COLUMN] column_old_name column_new_name column_dataType
   [COMMENT column_comment]
   [FIRST | AFTER column_name];
```
And reports `ParseException: Renaming column is not supported in Hive-style ALTER COLUMN, please run RENAME COLUMN instead.` because `TYPE` is parsed as a new name.